ERCS/erc-7066.md

```diff
@@ -4,7 +4,7 @@ title: Lockable Extension for ERC-721
 description: Interface for enabling locking of ERC-721 using locker and approved
 author: Piyush Chittara (@piyush-chittara), StreamNFT (@streamnft-tech), Srinivas Joshi (@SrinivasJoshi)
 discussions-to: https://ethereum-magicians.org/t/eip-7066-lockable-extension-for-erc721/14425
-status: Review
+status: Final
```

Comment:
You'll need to begin a Last Call period and add a `last-call-deadline` before you can go to Final. The Last Call status gives any potential reviewers notice that the proposal is about to go to Final.

 ```suggestion
status: Last Call
last-call-deadline: 2023-11-20
```